pages/_app.jsx

```diff
@@ -30,7 +36,7 @@ export default function App({ Component, pageProps }) {
         return game;
       })
     );
-    router.push(`/${gameToEdit.id}`);
+    router.push(`/${gameToEdit.id}`, { scroll: false });
```

Comment:
As we talked about after the checkout, you can remove this line of code and move the router.push to a different location of your application so that it will only be executed, whenever the user is on the edit page.

```suggestion
```